CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 ## 3.39.0 (Unreleased)
 
+IMPROVEMENTS:
+* compute: added `storage_locations` field to `google_compute_snapshot` ([#7201](https://github.com/hashicorp/terraform-provider-google/pull/7201))
+* compute: added `kms_key_service_account`, `kms_key_self_link ` fields to `snapshot_encryption_key` field in `google_compute_snapshot`
+* compute: added `source_disk_encryption_key.kms_key_service_account` field to `google_compute_snapshot`
```

Comment:
This entry and the one above it don't have the links to the PR after them - I think they should be the same as the first one, they were all done together.